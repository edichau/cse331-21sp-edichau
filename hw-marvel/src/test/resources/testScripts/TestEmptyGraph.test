# Test loading a graph of with empty csv file

LoadGraph heroes staffSuperheroes.csv