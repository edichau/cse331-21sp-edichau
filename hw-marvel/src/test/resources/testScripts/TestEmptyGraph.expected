# Test loading a graph of with empty csv file

loaded graph heroes